maintenance/wikia/jwplayermigration/migrateOoyalatoJWPlayer.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * Add JWPlayer mediaId to wgFeaturedVideo
+ *
+ */
+require_once( dirname( __FILE__ ) . '/../../Maintenance.php' );
+
+class MigrateOoyalaVideos extends Maintenance {
+
+	protected $dryRun = true;
+
+	protected $map = [];
+
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = "Migrate videos";
+		$this->addOption( 'dry-run', 'Dry run mode', false, false, 'd' );
+
+		$videos = array_map( 'str_getcsv', file( 'output.csv' ) );
+		$map = [];
+
+		foreach ( $videos as $video ) {
+			$map[$video[1]] = $video[0];
+		}
+
+		$this->map = $map;
+	}
+
+
+	public function execute() {
+		$this->dryRun = $this->hasOption( 'dry-run' );
+		$this->output( "Start \n" );
+
+		//$this->doUpdate( 1689, 'wgArticleVideoFeaturedVideos' );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)

[project-id]: wikia:app